docs/package-lock.json

```diff
@@ -1,4353 +1,8 @@
 {
   "name": "fluidframework-docs",
   "version": "0.25.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
I say leave this in v2 (npm7) format. I was going to bump the CI build to match. But if this is better for some reason go ahead.